github.js

```diff
@@ -1,14 +1,7 @@
 import got from 'got';
 import race from 'promise-rat-race';
 
-function getChangelog({
-  githubRepo = {
-    user: '',
-    project: '',
-    path: '',
-  },
-  gitHead = 'master',
```

Comment:
For reference, this is only called if gitHead key is not defined, not well null is passed.